pallets/parachain-staking/src/lib.rs

```diff
@@ -1745,26 +1744,29 @@ pub mod pallet {
 				}
 			};
 			let collator_fee = <CollatorCommission<T>>::get();
-			for (val, pts) in <AwardedPts<T>>::drain_prefix(round_to_payout) {
+			for (collator, pts) in <AwardedPts<T>>::drain_prefix(round_to_payout) {
```

Comment:
```suggestion
			for (author, pts) in <AwardedPts<T>>::drain_prefix(round_to_payout) {
```
I agree though, not a fan of `val`

Reply:
`author` works for me, `collator` is a bit long :)